docs/reference/node-feature-client-reference.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Node Feature client cmdline reference"
+layout: default
+sort: 8
+---
+
+# Commandline flags of nfd client
+{: .no_toc}
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+---
+
+To quickly view available command line flags execute `nfd --help`.
+
+### -h, --help
+
+Print usage and exit.
+
+## Compat
```

Comment:
when input the parameter, it's "compat", it's better to to use the term same as that in command line. the next page kubectl plugin style is not very good

Reply:
I fixed the name issue. 

> the next page kubectl plugin style is not very good

Could you elaborate on this? I didn't quite understand the comment.

Reply:
Discussed internally.